### PR TITLE
feat: added OTHER_INFO from tprek to venue pages

### PR DESCRIPTION
**Pull request by HeikkM**
LIIKUNTA-442 
Added /n handling also to venue description by adding white-space:pre-line to the scss file. 
Added translations for "Paikan lisätiedot" -title.

cs9szK8Rso

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr299.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr299.dev.hel.ninja :laughing::tada::tada::tada:

File: packages/common-i18n/src/locales/fi/venue.json [hunk 1]
```diff
@@ -6,7 +6,11 @@
     "title": "Jaa liikunta"
   },
   "description": {
-    "title": "Kuvaus"
+    "title": "Kuvaus",
+    "titleOtherInfo": "testing"
```
Comment (nikomakela) [inline packages/common-i18n/src/locales/fi/venue.json:10]: shouldn 't be here?

File: apps/sports-helsinki/src/domain/venue/venueOtherInfo/venueOtherInfo.module.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+.venueOtherInfo {
+  padding: 0;
+  font-size: var(--fontsize-body-l);
+
+  display: grid;
+  grid-template-columns: 1fr;
+
+  @include respond_above(m) {
+    // padding: var(--spacing-xl);
+  }
```
Comment (nikomakela) [inline apps/sports-helsinki/src/domain/venue/venueOtherInfo/venueOtherInfo.module.scss:12]: no need for anybcommented lines here. Should this then be removed?

**Review by nikomakela — APPROVED** [review]

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Events-Helsinki branch is deployed to platta: https://tapahtumat-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://harrastukset-pr299.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://tapahtumat-pr299.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**EVENTS GRAPHQL PROXY is deployed to platta: https://events-graphql-proxy-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**VENUE GRAPHQL PROXY is deployed to platta: https://venue-graphql-proxy-pr299.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr299.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**